CMakeLists.txt

```diff
@@ -16,10 +16,24 @@
 # under the License.
 
 cmake_minimum_required(VERSION 3.16.0)
+
+string(TIMESTAMP DT %Y%m%d UTC)
+string(TIMESTAMP HHMM %H%M UTC)
+configure_file(version.cfg.in ${CMAKE_CURRENT_SOURCE_DIR}/version.cfg @ONLY)
+file(STRINGS version.cfg BASE_VERSION)
+
 project(DataSketches
-        VERSION 3.6.0
+        VERSION ${BASE_VERSION}
         LANGUAGES CXX)
 
+#get_cmake_property(_variableNames VARIABLES)
```

Comment:
do we need to keep this commented-out section?

Reply:
Removed -- thanks.